src/DashboardClient.php

```diff
@@ -0,0 +1,149 @@
+<?php
+
+namespace Lullabot\Parsely;
+
+use GuzzleHttp\ClientInterface;
+use GuzzleHttp\HandlerStack;
+use Psr\Http\Message\RequestInterface;
+
+class DashboardClient implements ClientInterface
+{
+    /**
+     * The root URL for dashboard requests.
+     *
+     * @see https://www.parse.ly/help/integration/trigger-crawl/
+     * @see https://www.parse.ly/help/integration/crawler/#how-do-i-find-pages-that-have-not-been-crawled-correctly
+     */
+    const ROOT_URL = 'https://dash.parsely.com/';
+
+    /**
+     * The underlying HTTP client.
+     *
+     * @var \GuzzleHttp\ClientInterface
+     */
+    protected $client;
+
+    /**
+     * @var string
+     */
+    private $apikey;
+
+    /**
+     * @var string
+     */
+    private $secret;
+
+    /**
+     * Client constructor.
+     *
+     * @param \GuzzleHttp\ClientInterface $client The underlying HTTP client to use for requests.
+     * @param string                      $apikey
+     * @param string                      $secret
+     */
+    public function __construct(ClientInterface $client, string $apikey, string $secret)
+    {
+        $this->client = $client;
+        $this->apikey = $apikey;
+        $this->secret = $secret;
+    }
+
+    /**
+     * Get the default Guzzle client configuration array.
+     *
+     * @param mixed $handler (optional) A Guzzle handler to use for requests.
+     *
+     * @return array An array of configuration options suitable for use with Guzzle.
+     */
+    public static function getDefaultConfiguration($handler = null)
+    {
+        $config = [
+      'base_uri' => self::ROOT_URL,
```

Comment:
Huh, I wonder why this misindentation passed CS checks.

Reply:
🤷‍♂️ 

It was actually failing a couple commits before I got fed up and ran the same `vendor/bin/php-cs-fixer` command as the job was, just without `--dry-run`. I'll give it one more go.